common/beerocks/bcl/include/bcl/son/son_wireless_utils.h

```diff
@@ -154,6 +154,8 @@ class wireless_utils {
     static std::string wsc_to_bwl_authentication(WSC::eWscAuth authtype);
     static std::string wsc_to_bwl_encryption(WSC::eWscEncr enctype);
     static beerocks::eBssType wsc_to_bwl_bss_type(WSC::eWscVendorExtSubelementBssType bss_type);
+    static std::list<uint8_t> string_to_wsc_oper_class(const std::string &operating_class);
```

Comment:
```suggestion
    static std::vector<uint8_t> string_to_wsc_oper_class(const std::string &operating_class);
```
[SL.con.2: Prefer using STL vector by default unless you have a reason to use a different container](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#slcon2-prefer-using-stl-vector-by-default-unless-you-have-a-reason-to-use-a-different-container)

In this case it could even be an array, but it doesn't matter much.

Reply:
Decided to keep it as is because changing operating class from the list to vector can cause a lot of changes and it is not a part of this issue, i propose to create issue like refactoring - sBssInfoConf structure, where the operating class list will be changed on the vector.